RealifeTech-CoreSDK/APILayer/Util/String+RequestIdentifier.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  String+RequestIdentifier.swift
+//  RealifeTech-CoreSDK
+//
+//  Created by Mickey Lee on 05/03/2021.
+//
+
+import Foundation
+import CryptoKit
+
+extension String {
+
+    var md5: String {
+        let digest = Insecure.MD5.hash(data: self.data(using: .utf8) ?? Data())
```

Comment:
Should we look to improve this at some point?

Reply:
I haven't raised or seen the ticket for changing it to `sha256` but I can change it in this PR

Reply:
I would raise a ticket @mickeylee25 . That way you can get this PR in.

Reply:
I just committed the changes of SHA256 to this PR.